lib/detector.ts

```diff
@@ -15,6 +15,7 @@ const firstBytes: { [byte: number]: imageType } = {
   0x69: 'icns',
   0x89: 'png',
   0xff: 'jpg',
+  ["<".charCodeAt(0)]: 'svg',
```

Comment:
```suggestion
  0x3c: 'svg',
```

How about keeping this consistent with the others by using hex?